File: solutions/3172-divisible-and-non-divisible-sums-difference/solution.js
/**
 * @param {number} n
 * @param {number} m
 * @return {number}
 */
var differenceOfSums = function(n, m) {
    let num1 = 0;
    let num2 = 0;
    let i = 1;
    for (;i<=n;i++){
        if(i%m !== 0){
            num1+=i
        }
        else{
            num2+=i
        }
    }
    return num1 - num2;
};

File: solutions/3172-divisible-and-non-divisible-sums-difference/solution.cpp
class Solution {
public:
    int differenceOfSums(int n, int m) {
        int num1 = 0;
        int num2  = 0;
        int i = 1;
        for(;i<=n;i++){
            if(i % m != 0){
                num1+=i;
            }
            else{
                num2+=i;
            }
        }    
        return num1-num2;  
    }
};
